src/components/overview/Overview.tsx

```diff
@@ -0,0 +1,105 @@
+'use client';
+
+import { EmbeddedResource, GeoJSONResource, JSONResource } from '@/schemas/configuration-schema';
+
+import React from 'react';
+// eslint-disable-next-line import/named
+import { TableColumn } from 'react-data-table-component';
+import { createSharedPathnamesNavigation } from 'next-intl/navigation';
+import dynamic from 'next/dynamic';
+import { getColorForResourceType } from '@/colors';
+import { locales } from '@/locales';
+import { replaceWhitespaceInString } from '@/utils/stringutils';
+import { useTranslations } from 'next-intl';
+
+// Avoid hydration error inside Table pagination https://stackoverflow.com/questions/77763766/next-js-hydration-error-with-shadcn-dialog-component
+const DataTable = dynamic(() => import('react-data-table-component'), { ssr: false });
+
+const { Link } = createSharedPathnamesNavigation({
+  locales: [...locales.values()],
+});
+
+// TODO: We need a way to select the subcategory: Either in the navigation, or inside of a selector component at the top of the overview
```

Comment:
Agree. I wouldn't do it inside the navigation as it could become overloaded quickly. See my ideas in another comment. For now, it would be enough to have an overview page for every subcategory